doc/todo/RELEASE

```diff
@@ -25,238 +13,66 @@ Following shell expansions are used in this document:
- kdb --version | tee ~elektra/$VERSION/version
- kdb qt-gui
-
-Strace what tools are doing:
-
- strace -o ~elektra/$VERSION/mount.strace kdb mount file.ecf user/release_test
- strace -o ~elektra/$VERSION/file.strace kdb file user/release_test/b
- strace -o ~elektra/$VERSION/set.strace kdb set user/release_test/b
- strace -o ~elektra/$VERSION/get.strace kdb get user/release_test/b
- strace -o ~elektra/$VERSION/rm.strace kdb rm user/release_test/b
- strace -o ~elektra/$VERSION/umount.strace kdb umount user/release_test
-
-Run tests:
-
- KDB=kdb kdb run_all -v | tee ~elektra/$VERSION/run_all 2>&1
- KDB=kdb-full kdb-full run_all | tee ~elektra/$VERSION/run_all_full 2>&1
-
-Install libelektra-test of the previous version and run again:
-
- CHECK_VERSION=NO kdb run_all -v > ~elektra/$VERSION/run_all_abi 2>&1
-
-
-== Test external tools ==
-
-
-Check if tools based on Elektra still do:
- dq
-
-Run unit-tests in-source again (because some use installed Elektra):
- make run_all
-
-
-== Test external test suites ==
-
-#https://github.com/oyranos-cms/oyranos/blob/master/src/tests/test2.cpp#L185
-#https://github.com/oyranos-cms/oyranos/blob/master/src/liboyranos_config/oyranos_elektra.c#L195
-
-git clone git@github.com:oyranos-cms/oyranos.git
-cd oyranos
-mkdir build
-cd build
-cmake .. # might fail, simply comment out missing libs
-make -j5 # not needed
-make test2 # or make test-2 for the most of the tests
-src/tests/test2 Elektra
-
-
-== Documentation ==
-
-And the API docu:
- mkdir ~elektra-doc/api/$VERSION/
- cd ~build && cmake -DBUILD_PDF=ON . && rm -rf doc && make html man && cp -ra doc/html doc/latex doc/man ~elektra-doc/api/$VERSION/
-
-Symlink current to latest version and add everything:
- cd ~elektra-doc/api/ && rm current && ln -s $VERSION current && git add current $VERSION && git commit -a -m "$VERSION Release"
-
-
-
+- https://build.libelektra.org/job/libelektra/job/master/ must pass
+- Trigger release pipeline with dry run
+- Inspect the logs from the pipeline
+  - [ ] Error logs (*.error) must be empty
+  - [ ] Check in the (test-*) folders if all tests passed
+  - [ ] Check if version is correct in the version file
+  - [ ] Check if the debian package build log was successfull (debian/<DEBIAN_CODENAME>/elektra_*.buildinfo)
+- Check if all artifacts are present
+  - [ ] Check debian/<DEBIAN_CODENAME> if all debian packages were built
+  - [ ] Unarchive the elektra tarball and briefly verify if the correct version was archived
+- Inspect the changes to the libelektra git repository (*.bundle)
+  Inspect the git log of the master and debian branch with: git log -p --author="jenkins-release"
+  Master:
+  - [ ] Check diff of commit "Regenerate dot of plugins for release <VERSION>" and if 
+        doc/images/plugins.pdf was generated correctly (Only if plugins were modified)
+  - [ ] Check diff of commit "Update contract for release <VERSION>": Plugin tags in the plugin 
+        README.md should be sorted (Only if tags are not sorted)
```

Comment:
```suggestion
        README.md should be sorted
```

The tool also adds/removes tags in some situations.

Reply:
The tool only sorts tags if no arguments are provided and since there weren't arguments mentioned in the manual release steps I though the tool should be called without arguments. Should I use the auto heuristic of this tool?

> ... If no action is given it will sort the Tags, for the specified plugins or all if no PLUGINS are specified

Reply:
Thank you for reminding me about this! So there are actually two separate things:

- a manual invocation with arguments: #3553
- an automated invocation without arguments (as you wrote here)

Reply:
Is there a reason why the automated invocation of `update-info-status` shouldn't be called with the auto heuristic? I think the manual invocation can actually be omitted if we use the auto heuristic in the pipeline, since a manual invocation to change the plugin tags shouldn't actually be necessary?

Reply:
Such heuristics look to me as they will always need a manual check. Heuristics, by definition, might get things wrong.
If we really want to automate such things, then the build server should probably open a PR to show the changes.
It looks to me out-of-scope for your work. The benefit to automatize this is also quite small.